crates/sui-framework/sources/Governance/Genesis.move

```diff
@@ -20,6 +20,9 @@ module Sui::Genesis {
     /// Initial value of the upper-bound on the number of validators.
     const INIT_MAX_VALIDATOR_COUNT: u64 = 100;
 
+    /// Initial storage gas price
+    const INIT_STORAGE_GAS_PRICE: u64 = 1;
```

Comment:
IIRC we don't store the gas schedule (the gas cost of each bytecode instruction) on-chain. Wondering why we are storing the storage price on-chain?

Reply:
This is the storage gas price that will be used to transfer to storage fund https://docs.google.com/document/d/19q9IbTaUkQ3R6vwTMD2CGSAixkkGAmoWZDeS0DOrgqE/edit#heading=h.eazvgbbetjv8